ERRORS
Error:(<type 'exceptions.AttributeError'>, AttributeError("'NoneType' object has no attribute 'keys'",), <traceback object at 0x7f92cf3a5e18>)
Error:(<type 'exceptions.AttributeError'>, AttributeError("'NoneType' object has no attribute 'keys'",), <traceback object at 0x7f92cf428a28>)
Error:(<type 'exceptions.AttributeError'>, AttributeError("'NoneType' object has no attribute 'keys'",), <traceback object at 0x7f92cd875bd8>)
Error:(<type 'exceptions.AttributeError'>, AttributeError("'NoneType' object has no attribute 'keys'",), <traceback object at 0x7f92cd956f38>)
Error:(<class 'requests.exceptions.ConnectionError'>, ConnectionError(MaxRetryError("HTTPConnectionPool(host='api.eventful.com', port=80): Max retries exceeded with url: /json/events/tags/list?id=E0-001-090689990-3&app_key=SvV5KrR9B3PMS9HB (Caused by <class 'socket.error'>: [Errno 111] Connection refused)",),), <traceback object at 0x7f92cd90def0>)
Error:(<class 'requests.exceptions.ConnectionError'>, ConnectionError(MaxRetryError("HTTPConnectionPool(host='api.eventful.com', port=80): Max retries exceeded with url: /json/events/search?app_key=SvV5KrR9B3PMS9HB&image_sizes=small%2Cmedium%2Clarge&page_number=204&page_size=100&date=all&include=categories%2Clinks%2Csubcategories%2Cpopularity%2Ctickets%2Cprice&location=US (Caused by <class 'socket.error'>: [Errno 111] Connection refused)",),), <traceback object at 0x7f92cda9ab00>)
Error:(<class 'requests.exceptions.ConnectionError'>, ConnectionError(MaxRetryError("HTTPConnectionPool(host='api.eventful.com', port=80): Max retries exceeded with url: /json/events/search?app_key=SvV5KrR9B3PMS9HB&image_sizes=small%2Cmedium%2Clarge&page_number=205&page_size=100&date=all&include=categories%2Clinks%2Csubcategories%2Cpopularity%2Ctickets%2Cprice&location=US (Caused by <class 'socket.error'>: [Errno 111] Connection refused)",),), <traceback object at 0x7f92cd76e950>)
Error:(<class 'requests.exceptions.ConnectionError'>, ConnectionError(MaxRetryError("HTTPConnectionPool(host='api.eventful.com', port=80): Max retries exceeded with url: /json/events/search?app_key=SvV5KrR9B3PMS9HB&image_sizes=small%2Cmedium%2Clarge&page_number=206&page_size=100&date=all&include=categories%2Clinks%2Csubcategories%2Cpopularity%2Ctickets%2Cprice&location=US (Caused by <class 'socket.error'>: [Errno 111] Connection refused)",),), <traceback object at 0x7f92cd8d2128>)
Error:(<class 'requests.exceptions.ConnectionError'>, ConnectionError(MaxRetryError("HTTPConnectionPool(host='api.eventful.com', port=80): Max retries exceeded with url: /json/events/search?app_key=SvV5KrR9B3PMS9HB&image_sizes=small%2Cmedium%2Clarge&page_number=207&page_size=100&date=all&include=categories%2Clinks%2Csubcategories%2Cpopularity%2Ctickets%2Cprice&location=US (Caused by <class 'socket.error'>: [Errno 111] Connection refused)",),), <traceback object at 0x7f92cd8d2e60>)
Error:(<class 'requests.exceptions.ConnectionError'>, ConnectionError(MaxRetryError("HTTPConnectionPool(host='api.eventful.com', port=80): Max retries exceeded with url: /json/events/search?app_key=SvV5KrR9B3PMS9HB&image_sizes=small%2Cmedium%2Clarge&page_number=208&page_size=100&date=all&include=categories%2Clinks%2Csubcategories%2Cpopularity%2Ctickets%2Cprice&location=US (Caused by <class 'socket.error'>: [Errno 111] Connection refused)",),), <traceback object at 0x7f92cdae0908>)
Error:(<class 'requests.exceptions.ConnectionError'>, ConnectionError(MaxRetryError("HTTPConnectionPool(host='api.eventful.com', port=80): Max retries exceeded with url: /json/events/search?app_key=SvV5KrR9B3PMS9HB&image_sizes=small%2Cmedium%2Clarge&page_number=209&page_size=100&date=all&include=categories%2Clinks%2Csubcategories%2Cpopularity%2Ctickets%2Cprice&location=US (Caused by <class 'socket.error'>: [Errno 111] Connection refused)",),), <traceback object at 0x7f92cdae0710>)
Error:(<class 'requests.exceptions.ConnectionError'>, ConnectionError(MaxRetryError("HTTPConnectionPool(host='api.eventful.com', port=80): Max retries exceeded with url: /json/events/search?app_key=SvV5KrR9B3PMS9HB&image_sizes=small%2Cmedium%2Clarge&